GNUmakefile

```
@@ -30,6 +30,10 @@ gomod: ## Ensure chainlink's go dependencies are installed.
 gomodtidy: gomods ## Run go mod tidy on all modules.
 	gomods tidy
 
+.PHONY: gomodrequiredupdater
+gomodrequiredupdater: ## Update go.mod files containing certain required dependencies to use latest psuedo-versions from trunk.
+	cd tools/gomod-required-updater && go run cmd/gomod-required-updater/main.go -update-org-modules -root ../..
```

If there is a good reason to do the go.mod discovery inside of the tool, then nbd, but otherwise consider simplifying it to be single-module, and invoking via gomods:
```suggestion
	gomods -u -go run cmd/gomod-required-updater
```

---

Converted to `gomods` in 99ad50b391636321f4960db9e6db4184fb114ff1.